mlir/include/mlir/Dialect/DLTI/DLTI.h

```diff
@@ -124,6 +126,150 @@ class DataLayoutSpecAttr
   static constexpr StringLiteral name = "builtin.data_layout_spec";
 };
 
+//===----------------------------------------------------------------------===//
+// TargetSystemDescSpecAttr
+//===----------------------------------------------------------------------===//
+
+/// A system description attribute is a list of device descriptors, each
+/// having a unique device ID
+class TargetSystemDescSpecAttr
```

Comment:
As mentioned in the other PR: I don't quite get why these can't be defined in ODS, so please define them in ODS or document why they cannot be.

Reply:
Fixed it.